### PR TITLE
chore: add delete modules in npm script

### PR DESCRIPTION
**Pull request by thorseraq**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "test:unit": "playwright test --config=playwright.config.unit.ts",
     "postinstall": "husky install",
     "notify": "node --experimental-modules scripts/notify.mjs",
-    "update:blocksuite": "pnpm i --filter @affine/app --filter @affine/datacenter @blocksuite/blocks@nightly @blocksuite/store@nightly && pnpm i --filter @affine/app @blocksuite/editor@nightly"
+    "update:blocksuite": "pnpm i --filter @affine/app --filter @affine/datacenter @blocksuite/blocks@nightly @blocksuite/store@nightly && pnpm i --filter @affine/app @blocksuite/editor@nightly",
+    "delete:node_modules": "find . -name \"node_modules\" -exec rm -rf '{}' +"
```
Comment (himself65) [inline package.json]: could add a shell script instead of in package.json?

**Review by Himself65 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (thorseraq) [reply package.json]: Make sense, how about changes in https://github.com/toeverything/AFFiNE/pull/900/commits/ff772e6024089a3f91b804ed673d64f0c403e6d3

**Review by lawvs — CHANGES_REQUESTED** [review]
It's should not be a common operation

**Comment by thorseraq** [comment]
> It's should not be a common operation

Are you suggesting close it? Add an tool doesn't mean it's a common operation though

**Pull request closed without merging**